### PR TITLE
Issue #387 Add Un-install instructions

### PR DESCRIPTION
**Pull request by BrunoVernay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,12 @@ You can now install the latest beta version of minishift.
   $ brew cask install minishift-beta
 ```
 
+### Un-installing Minishift
+
+.1 Try to delete the Minishift VM: `minishift delete`
```
Comment (budhrg) [inline README.md]: This is not rendering as expected. See below
screenshot from 2017-02-15 00-33-48

File: README.md [hunk 2]
```diff
@@ -100,6 +100,12 @@ You can now install the latest beta version of minishift.
...
+.1 Remove the folders: `~/.minishift` and `~/.kube`
+.1 Check that there are no remaining Minishift VM. Use KVM (virsh), VirtualBox or Xhyve depending on your driver.
```
Comment (LalatenduMohanty) [inline README.md]: I do not think we have a way to check if there is an existing VM in Xhyve. So we should add a note about Xhyve that user can not findout if the VM is present in Xhyve.

**Comment by Preeticp** [comment]
There is a section on Minishift Delete in Using.md, (https://github.com/minishift/minishift/blob/master/docs/using.md#deleting-openshift) can we add this information there?

**Comment by hferentschik** [comment]
I will take care of this as part of pull request #400 and issue #378

**Comment by hferentschik** [comment]
@BrunoVernay, merged thanks. I added some more reference links as well. 

Thanks again for your help. Let us know if you find more :-)

**Comment by BrunoVernay** [comment]
Yes I just saw, too late, that the changes had been already done.
There is just a typo: "conform" instead of "confirm"

**Comment by hferentschik** [comment]
Ok, thanks

**Comment by hferentschik** [comment]
Fixed :-)

**Pull request closed without merging**